packages/wallet/package.json

```diff
@@ -54,8 +56,10 @@
     "@fuel-ts/signer": "workspace:*",
     "@fuel-ts/transactions": "workspace:*",
     "@fuel-ts/utils": "workspace:*",
+    "@fuels/vm-asm": "0.34.1",
```

Comment:
These packages don't seem to be necessary:
 - `@fuel-ts/crypto`
 - `@fuels/vm-asm`

Please remove them.

Reply:
Never mind, both are used.

I mixed things up, my bad.